plutus-core/untyped-plutus-core/test/Evaluation/Golden/prod1.plc.golden

```diff
@@ -1 +0,0 @@
-(Right (prod (con integer 11) (con integer 12)))
```

Comment:
oh wow, this was my bad, these really were leftover golden files from a very old version of SOPs!